src/protagonist/API.Client/DlcsClient.cs

```diff
@@ -81,9 +81,17 @@ public class DlcsClient : IDlcsClient
     public async Task<Space?> GetSpaceDetails(int spaceId)
     {
         var url = $"customers/{currentUser.GetCustomerId()}/spaces/{spaceId}";
-        var response = await httpClient.GetAsync(url);
-        var space = await response.ReadAsHydraResponseAsync<Space>(jsonSerializerSettings);
-        return space;
+        try
+        {
+            var response = await httpClient.GetAsync(url);
+            var space = await response.ReadAsHydraResponseAsync<Space>(jsonSerializerSettings);
+            return space;
+        }
+        catch (Exception ex)

```

Comment:
I don't think the `ex` is used - might be better to remove it, or pass it into the logger with `logger.LogError(ex, "Failed to retrieve space {SpaceId}", spaceId);`